src/main/webapp/app/shared/feature-flags.ts

```diff
@@ -0,0 +1 @@
+export const STOP_REVIEW_IF_CORE_SUBMISSION_FAILS: boolean = false;
```

Comment:
Should this be in appConfig.ts?

Reply:
I think you could make an argument either way for this. If you feel like you'll want to switch the behavior without creating a new build, then yes. If you feel like we'll make a code change first then rolls this out, then it doesn't make a different. This is entirely for myself to track the places I need to rollback.

Reply:
Let's move this to AppConfig to consolidate.